lib/helpers.nix

```diff
@@ -1,7 +1,7 @@
 {
   lib,
-  pkgs,
-  _nixvimTests,
+  pkgs ? pkgs.lib,
```

Comment:
Why does `pkgs` default to `pkgs.lib` ?

Reply:
Throughout docs, wrappers, etc, if we do not have a better way of getting `lib`, we get it from `pkgs.lib`.

This is somewhat redundant in most cases, but it matches what is done in `lib/default.nix`.

Reply:
Sure, but shouldn't it be `lib ? pkgs.lib` ?

Reply:
I'm blind, you're right lol